gradle/libs.versions.toml

```diff
@@ -2,17 +2,24 @@
 # libraries
 junit = "4.13.2"
 opentest4j = "1.3.0"
+hamcrest = "2.2"
+# Has to be in sync with IntelliJ Platform
+composeuitest="1.8.0-alpha04"
+jewelstandalone="0.29.0-251.27828"
```

Comment:
0.29 was released in 251.4.1, you probably want to bump the IJP version in `gradle.properties`